docs/fundamentals/code-analysis/categories.md

```diff
@@ -10,19 +10,19 @@ helpviewer_keywords:
 
 Each code analysis rule belongs to a category of rules. For example, design rules support adherence to the .NET design guidelines, and security rules help prevent security flaws. You can [configure the severity level](configuration-options.md#scope) for an entire category of rules. You can also [configure additional options](code-quality-rule-options.md#category-of-rules) on a per-category basis.
 
-The following table shows the different code analysis rule categories and provides a link to the rules in each category.
+The following table shows the different code analysis rule categories and provides a link to the rules in each category. It also lists the configuration value to use in an EditorConfig file to bulk-configure rule severity on a per-category basis. For example, to set the severity of security rule violations to be errors, the EditorConfig entry is `dotnet_analyzer_diagnostic.category-Security.severity = error`.
```

Comment:
Should I list CodeQuality (and its corresponding config value `dotnet_analyzer_diagnostic.category-CodeQuality.severity`) in the table? If so, what exactly does it encompass?

Reply:
I believe the `dotnet_analyzer-diagnostic.category-XXX.severity` doesn't apply to rules that are disabled by default. This information could be noted somewhere else. But I think it's helpful to include it here (or a link to it). I'd also include a link or a note to `<AnalysisModeCategory>` (which as far as I understand applies to ALL rules, whether or not they're disabled by default)

Reply:
> Should I list CodeQuality (and its corresponding config value `dotnet_analyzer_diagnostic.category-CodeQuality.severity`) in the table? If so, what exactly does it encompass?

I think it should be included. Note that not all IDExxxx have "Style" category. Examples:

- [IDE0064](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0064)
- [IDE0076](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0076)
- [IDE0051](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0051)
- [IDE0079](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0079) (this one looks like it's incorrectly documented as "Style". Opened https://github.com/dotnet/docs/pull/26077 for this one.